### PR TITLE
[PATCH v4] Include ipsec examples into 'make check'

### PR DESCRIPTION
**Pull request by psavol**
IPSec example application have not been part of make check. Fix various issues found during testing and run examples as part of 'make check'.

### CODE REVIEW

File: example/ipsec/odp_ipsec.c [hunk 1]
```diff
@@ -241,7 +241,7 @@ typedef odp_queue_t (*queue_create_func_t)
 typedef odp_event_t (*schedule_func_t) (odp_queue_t *);
 
 static queue_create_func_t queue_create;
-static schedule_func_t schedule;
+static schedule_func_t schedule_fn;
```
Comment (lumag) [inline example/ipsec/odp_ipsec.c]: This rename is irrelevant to `odp_crypto_op()/odp_crypto_op_enq()` return codes and should be split to a separate patch.

Comment (psavol) [reply example/ipsec/odp_ipsec.c]: Renames moved to separate patch in v3

Comment (lumag) [reply example/ipsec/odp_ipsec.c]: Thank you!

Comment (psavol) [reply example/ipsec/odp_ipsec.c]: I could add your reviewed-by tag and merge this PR. Have you reviewed the entire set, or only this patch?

Comment (lumag) [reply example/ipsec/odp_ipsec.c]: Yes, please. I have reviewed whole series.

Reviewed-by: Dmitry Eremin-Solenikov
